profiling/__init__.py

```diff
@@ -6,7 +6,7 @@
 try:
     from cStringIO import StringIO
 except ImportError:
-    from StringIO import StringIO
+    from io import StringIO
```

Comment:
Can we still retain support for python 2.x by using the following code ?
```python
try:
    from cStringIO import StringIO	
except ImportError:	
    try: 
       from StringIO import StringIO
    except importError:
       from io import StringIO

Reply:
Its nice to have support. We can commit these changes

Reply:
Can you  pls incorporate my comment into your PR ? Then we can merge immediately. Thank you

Reply:
Changes incorporated. Please merge it